static/app/components/timeSince.tsx

```diff
@@ -82,7 +84,10 @@ function TimeSince({
   const dateObj = getDateObj(date);
   const user = ConfigStore.get('user');
   const options = user ? user.options : null;
-  const format = options?.clock24Hours ? 'MMMM D, YYYY HH:mm z' : 'LLL z';
+  const tooltipFormat = tooltipShowSeconds
+    ? 'MMM D, YYYY h:mm:ss A z'
```

Comment:
Did you mean to change from `MMMM` to `MMM` here?

Reply:
yes! shorter months because `September` causes an overflow of the tooltip.

It would look like this with the longer month:
<img width="390" alt="Screen Shot 2022-09-26 at 11 22 44 AM" src="https://user-images.githubusercontent.com/187460/192353461-ecc77183-12a8-4430-ade9-8162692f81c9.png">